get_all_manifests.sh

```diff
@@ -11,6 +11,8 @@ REPO_LIST=(
     "kubeflow:v1.7-branch:components/notebook-controller/config:odh-notebook-controller/kf-notebook-controller"
     "kubeflow:v1.7-branch:components/odh-notebook-controller/config:odh-notebook-controller/odh-notebook-controller"
     "notebooks:main:manifests:notebooks"
+    "odh-model-controller:release-0.11:config:odh-model-controller"
+    "modelmesh-serving:release-0.11:opendatahub/manifests:model-mesh"
```

Comment:
ok, so we cannot merge this PR till  https://github.com/opendatahub-io/odh-model-controller/pull/92 and https://github.com/opendatahub-io/modelmesh-serving/pull/221 PRs are in

Reply:
think you can also update line 5

Reply:
https://github.com/opendatahub-io/odh-model-controller/pull/92 and https://github.com/opendatahub-io/modelmesh-serving/pull/221 are the PRs for the main branches to those repos, 
the PRs that are a prereq to this one are https://github.com/opendatahub-io/odh-model-controller/pull/93 and https://github.com/opendatahub-io/modelmesh-serving/pull/222 which are targeted at the release branches